### ISSUE TITLE
WuerstchenCombinedPipeline Import error in version 0.21.0

### ISSUE THREAD

**Issue by rupeshs** [huggingface/diffusers#5023]
### Describe the bug

Failed to import import `WuerstchenCombinedPipeline` from `diffusers.pipelines.wuerstchen`,
Tested on Google Colab

### Reproduction

import torch
from diffusers import AutoPipelineForText2Image

### Logs

```shell
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
/usr/local/lib/python3.10/dist-packages/diffusers/utils/import_utils.py in _get_module(self, module_name)
    683         try:
--> 684             return importlib.import_module("." + module_name, self.__name__)
    685         except Exception as e:

12 frames
ImportError: cannot import name 'WuerstchenCombinedPipeline' from 'diffusers.pipelines.wuerstchen' (/usr/local/lib/python3.10/dist-packages/diffusers/pipelines/wuerstchen/__init__.py)

The above exception was the direct cause of the following exception:

RuntimeError                              Traceback (most recent call last)
/usr/local/lib/python3.10/dist-packages/diffusers/utils/import_utils.py in _get_module(self, module_name)
    684             return importlib.import_module("." + module_name, self.__name__)
    685         except Exception as e:
--> 686             raise RuntimeError(
    687                 f"Failed to import {self.__name__}.{module_name} because of the following error (look up to see its"
    688                 f" traceback):\n{e}"

RuntimeError: Failed to import diffusers.pipelines.auto_pipeline because of the following error (look up to see its traceback):
cannot import name 'WuerstchenCombinedPipeline' from 'diffusers.pipelines.wuerstchen' (/usr/local/lib/python3.10/dist-packages/diffusers/pipelines/wuerstchen/__init__.py)
```

### System Info

Diffuser : v0.21.0 only
Environment: Google Colab, 
Python version: 3.10.12 (main, Jun 11 2023, 05:26:28) [GCC 11.4.0]
pytorch : 2.0.1+cu118

**Comment by jwooldridge234** [comment]
I'm also seeing this issue on my local machine (MPS).

**Comment by DN6** [comment]
@rupeshs Thanks for flagging. Looks like an issue with AutoPipeline when `transformers` is not installed. Could you try `pip install transformers` in your notebook and then run your code again?

**Comment by patrickvonplaten** [comment]
This should be fixed here: #5036

@DN6 we cannot assume that `transformers` is installed (it's on optional dependency). If it's not installed we should raise a nice error message

**Comment by rubensmau** [comment]
The same problem happens to me, even after installing transformers

**Comment by DN6** [comment]
@rubensmau we just did a patch release that should address the issue. Could you please upgrade your `diffusers` version and try running your code again?

**Comment by DN6** [comment]
@rupeshs  Could you please share a code snippet with the error? I'm running the following snippet and not seeing the error.
```
import torch
from diffusers import AutoPipelineForText2Image
```

**Comment by zba** [comment]
> `transformer`

edit to "transformers" please. you will keep another 5mins who pass through this error

**Comment by rupeshs** [comment]
@DN6  Please check this colab https://colab.research.google.com/drive/1Brinkt40qpPn6RTrwSNxmU4yvni8tJsf?usp=sharing

**Comment by DN6** [comment]
Hi @rupeshs for that specific example that you shared, you will need to install `transformers`. It is a required dependency for the  Wuerstchen pipeline and it won't run without it.  

As for this part 

```
from diffusers.pipelines.wuerstchen import DEFAULT_STAGE_C_TIMESTEPS
```
I'll look into how we might be able to better enable accessing constants defined in pipelines with lazy import enabled.

**Comment by rupeshs** [comment]
Hi @DN6 , BTW I have integrated diffuser Würstchen workflow to DiffusionMagic  https://github.com/rupeshs/diffusionmagic

**Comment by zba** [comment]
I ended to use diffusers frim github, because embed was not fixed in 0.21.2
